Levels/Earlytown/Earlytown.level

```diff
@@ -0,0 +1,263 @@
+EARLYTOWN is freely distributed under the following terms:
+Copyright (C) 2010 Edvin "Lego3" Linge, 2020 lloyd-lejack, 2024 Matroftt

```

Comment:
I don't believe Lloyd is part of this file. Matroftt's year is 2023 (also adjust the license.txt)